api/go.mod

```diff
@@ -19,8 +19,8 @@ require (
 	go.opentelemetry.io/otel/sdk v1.7.0
 	go.opentelemetry.io/otel/trace v1.7.0
 	go.opentelemetry.io/proto/otlp v0.16.0
-	golang.org/x/crypto v0.0.0-20220126234351-aa10faf2a1f8
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
+	golang.org/x/crypto v0.0.0-20220919173607-35f4265a4bc0
```

Comment:
Anything potentially problematic in the crypto/ssh diff between these versions?

Reply:
Not that I could spot (link for reference: https://cs.opensource.google/go/x/crypto;bpv=1)